### PR TITLE
Add '/server-error' route and view

### PR DESCRIPTION
**Pull request by nblair**
This pull request is intended to address issue #85. It adds a route/view that could be display for 500 class errors and service issues.

Example:

screen shot 2015-12-07 at 10 20 33 am

### CODE REVIEW

**Comment by timlevett** [comment]
:+1:

**Comment by nblair** [comment]
Any tests needed? I couldn't find any for `/access-denied` to model off of.

If we add an httpInterceptor that reroutes to this view, we'd want to test then.

**Comment by apetro** [comment]
Cf. [style guidance wrt coarse grained access denied|http://staging-2.uc.wisc.edu/my-uw/#coarse-grain-access].  I like that this looks similar.

File: src/main/webapp/portal/main/partials/server-error.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<frame-page app-header-title='{{NAMES.title}} - Service Temporarily Unavailable' 
+            app-header-collapse='true'
+            app-header-image='img/terrace.jpg'>
+  <h2 class='center'>We're sorry, but there was a problem with your request.</h2>
+  <p class='center'>
+    <i class='fa fa-exclamation-triangle fa-3x' style='color: #b70101;'></i>
```
Comment (apetro) [inline src/main/webapp/portal/main/partials/server-error.html]: Hardcoded color?  Could this be an alias?  Do we need CSS classes for error messaging?

**Comment by timlevett** [comment]
We currently don't have any tests for route logic as the apps overwrite the routes in most cases. Or were you saying you wanted to test the partial in some way?

File: src/main/webapp/portal/main/partials/server-error.html (hunk 1 above)
Comment (ahoffmann-wi) [inline src/main/webapp/portal/main/partials/server-error.html]: Agreed. `<style>` tags should be avoided and a class used.

Comment (nblair) [inline src/main/webapp/portal/main/partials/server-error.html]: It's pretty much a copy of the access-denied partial in the same folder, and has the same faults. I'd be happy to define a class and apply to both.

Comment (timlevett) [inline src/main/webapp/portal/main/partials/server-error.html]: :+1:

**Comment by paulerickson** [comment]
:+1: It's really content, not behavior in my opinion, so tests would belong in the downstream apps that decide when and why to route here.

**Comment by nblair** [comment]
OK to merge by me. We'd like to see a 1.3.4 release today if possible.

**Pull request merged**